shared/js/media/video_player.js

```diff
@@ -537,3 +577,15 @@ VideoPlayer.prototype.show = function() {
   // Call init() to show the poster
   this.controls.style.display = 'block';
 };
+
+VideoPlayer.instancesToLocalize = new WeakMap();
+
+navigator.mozL10n.ready(function() {
+  // Retrieve VideoPlayer instances by searching for container nodes.
+  for (var container of document.querySelectorAll('.video-player-container')) {
+    var instance = VideoPlayer.instancesToLocalize.get(container);
+    if (instance) {
+      instance.localize();
+    }
+  }
+});
```

Comment:
I think I will leave it at the bottom. There is so much nice documentation on top before the function declaration, and I feel like the localization bits just make that confusing.